package.json

```diff
@@ -70,28 +69,5 @@
       "type": "Apache-2.0",
       "url": "http://www.apache.org/licenses/LICENSE-2.0"
     }
-  ],
-  "jest": {
-    "collectCoverageFrom": [
-      "src/**",
-      "!src/index.js",
-      "!src/**/index.js",
-      "!src/configurations/**",
-      "!src/enums/**",
-      "!src/__tests__/test-data/**"
-    ],
-    "coverageThreshold": {
-      "global": {
-        "branches": 75,
-        "functions": 75,
-        "lines": 75,
-        "statements": 75
-      }
-    },
-    "coverageReporters": [
-      "cobertura",
-      "lcov",
-      "text"
-    ]
-  }
+  ]
```

Comment:
You should keep collectCoverageFrom in some form and coverageReporters. It is only necessary to remove coverageThreshold.

Reply:
Done